portfolio/src/main/webapp/script.js

```diff
@@ -47,6 +47,8 @@ function toggleVisibility (sectionToToggle) {
 
 async function fetchGreeting() {
   const response = await fetch('/data');
-  const greeting = await response.text();
-  document.getElementById('fetched-greeting').innerHTML = greeting;
+  const greetings = await response.json();
+  console.log(greetings);
+  const greetingNumber = Math.floor(Math.random() * 3);
```

Comment:
Oh also, let's use greetings.length instead of hard-coding "3" so that if the size ever changes, we don't have to worry about this.